mne/stats/cluster_level.py

```diff
@@ -488,13 +489,12 @@ def _pval_from_histogram(T, H0, tail):
 
     # from pct to fraction
     if tail == -1:  # up tail
-        pval = np.array([np.sum(H0 <= t) for t in T])
+        pval = np.array([np.mean(H0 <= t) for t in T])
     elif tail == 1:  # low tail
-        pval = np.array([np.sum(H0 >= t) for t in T])
+        pval = np.array([np.mean(H0 >= t) for t in T])
     else:  # both tails
-        pval = np.array([np.sum(abs(H0) >= abs(t)) for t in T])
+        pval = np.array([np.mean(abs(H0) >= abs(t)) for t in T])
 
-    pval = (pval + 1.0) / (H0.size + 1.0)  # the init data is one resampling
```

Comment:
I am just concerned by this. Do you guarantee that the non permuted stat value is in H0 ?

Reply:
Yes it's [in the code](https://github.com/mne-tools/mne-python/pull/4478/files#diff-23fe117cda04337c2d4a6c0fe30f0c76R825) and also [explicitly tested](https://github.com/mne-tools/mne-python/pull/4478/files#diff-750c85c929d3efd224d6c1a4c8147c15R500)